examples/Readme.md

```diff
@@ -1,38 +1,57 @@
 # Example Code
-
 This directory contains example models meant to test and demonstrate Mesa's features, and provide demonstrations for how to build and analyze agent-based models. For more information on each model, see its own Readme and documentation.
 
+## Models
+
+Classic models, some of which can be found in NetLogo's/MASON's example models.
+
+### bank_reserves
+A highly abstracted, simplified model of an economy, with only one type of agent and a single bank representing all banks in an economy.
+
 ### Color Patches
 A cellular automaton model where agents opinions are influenced by that of their neighbors. As the model evolves, color patches representing the prevailing opinion in a given area expand, contract, and sometimes disappear.
 
-### ConwaysGameOfLife
-
+### conways_game_of_life
 Implementation of [Conway's Game of Life](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life), a cellular automata where simple rules can give rise to complex patterns.
 
-### EpsteinCivilViolence
+### epstein_civil_violence
 Joshua Epstein's [model](http://www.uvm.edu/~pdodds/files/papers/others/2002/epstein2002a.pdf) of how a decentralized uprising can be suppressed or reach a critical mass of support.
 
-### Flockers
+### boid_flockers
 [Boids](https://en.wikipedia.org/wiki/Boids)-style flocking model, demonstrating the use of agents moving through a continuous space following direction vectors.
 
-### ForestFire
+### forest_fire
 Simple cellular automata of a fire spreading through a forest of cells on a grid, based on the NetLogo [Fire model](http://ccl.northwestern.edu/netlogo/models/Fire).
 
-### PD_Grid
+### hex_snowflake
+Conway's game of life on a hexagonal grid. TODO example/hex_snowflake/Readme.md needs to be updated.
```

Comment:
I think we should resolve this TODO within this PR or have an issue or other PR ready. I think this readme file is too visible for users to include TODO texts (which are for contributers and not users)